app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceItem.kt

```diff
@@ -1,8 +1,15 @@
 package com.example.keniphhomemaintenance.maintenance_screen
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "maintenance_item_table")
 data class MaintenanceItem(
+    @PrimaryKey(autoGenerate = true)
+    val maintenanceItemID: Int,
     val icon: Int,
     val title: String,
     val dueDate: String,
-    val location: String
+    val location: String,
+    val dwellingID: Int
```

Comment:
Kind of a weird thing to wrap your brain around at first, but generally in CS we don't use two-way relationships unless we have a good reason to. What I mean is that in this PR, we currently have a list of MaintenanceItems which is kept track of from the Dwelling, and then each MaintenanceItem points back to its Dwelling. 

The thing is, that second step (MaintenanceItem --> Dwelling) will probably never be used, because we wouldn't talk about a MaintenanceItem without the context of the Dwelling. In other words, if we are dealing with a MaintenanceItem, we already know which Dwelling it's associated with, because we'll be accessing the MaintenanceItem via the Dwelling.